miniwob/html/miniwob/enter-text-ocr.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Find Word Task</title>
```

Comment:
Nit: Update the title.

Reply:
:+1: -- not sure if we should include this at all, at least not in the same PR that puts the ground truth in JS variables.This `enter-text-ocr` is an additional task that requires entering text to help models learn OCR.